alacritty/src/config/monitor.rs

```diff
@@ -73,17 +88,15 @@ pub fn watch(mut paths: Vec<PathBuf>, event_proxy: EventProxy) {
             };
 
             match event {
-                DebouncedEvent::Rename(..) => continue,
-                DebouncedEvent::Write(path)
+                DebouncedEvent::Rename(_, path)
+                | DebouncedEvent::Write(path)
                 | DebouncedEvent::Create(path)
-                | DebouncedEvent::Chmod(path) => {
-                    if !paths.contains(&path) {
-                        continue;
-                    }
-
+                | DebouncedEvent::Chmod(path)
+                    if paths.contains(&path) =>
+                {
                     // Always reload the primary configuration file.
                     event_proxy.send_event(Event::ConfigReload(paths[0].clone()));
-                },
+                }
```

Comment:
```suggestion
                },
```

Reply:
It seems like the formatter is [complaining](https://builds.sr.ht/~undeadleech/job/568957#task-rustfmt-77) about this

Reply:
You're right. I'm really surprised by this since we have `match_block_trailing_comma = true` in our config.

I feel like this might be somewhat of a bug, because it doesn't happen without the `if`.